packages/aws-client/package.json

```diff
@@ -50,6 +50,7 @@
     "@aws-sdk/client-cloudwatch-events": "^3.447.0",
     "@aws-sdk/client-dynamodb": "^3.447.0",
     "@aws-sdk/client-dynamodb-streams": "^3.447.0",
+    "@aws-sdk/client-ec2": "^3.525.0",
```

Comment:
I suggest setting the client-ec2 version to 3.447.0 like the rest of the AWS SDK packages.

Reply:
Fixed in latest push.